Programs/_bootstrap_python.c

```diff
@@ -63,6 +63,15 @@ main(int argc, char **argv)
 {
     PyStatus status;
 
+    PyPreConfig preconfig;
+    PyPreConfig_InitIsolatedConfig(&preconfig);
+    // Force utf8 encoding
+    preconfig.utf8_mode = 1;
```

Comment:
You may add a comment explaining that the pre-initialization doesn't parse argv to keep the code simple. For example, ignore -E, -I and -X dev command line options.

Reply:
done